app/src/components/ArtifactComments.tsx

```diff
@@ -0,0 +1,95 @@
+import { Avatar, createStyles, Group, Paper, rem, Text, TypographyStylesProvider } from '@mantine/core';
+import { useHover } from '@mantine/hooks';
+import { useEffect, useState } from 'react';
+
+const useStyles = createStyles(theme => ({
+  card: {
+    borderRadius: 12,
+    border: `${rem(2)} solid ${theme.colors.gray[3]}`,
+    width: '800px'
+  },
+  comment: {
+    padding: `${theme.spacing.lg} ${theme.spacing.xl}`
+  },
+
+  body: {
+    borderRadius: 12,
+    paddingLeft: rem(54),
+    paddingTop: theme.spacing.sm,
+    fontSize: theme.fontSizes.sm
+  },
+
+  content: {
+    '& > p:last-child': {
+      marginBottom: 0
+    },
+    width: '800px'
+  }
+}));
+
+interface CommentHtmlProps {
+  date: string | undefined;
+  body: string | undefined;
+  author: string | undefined;
+}
```

Comment:
I would also maybe change this interface's name to be `ArtifactCommentsProps` so that it matches the `ArtifactComments` function. This means we will have to change the interface name in the following file, but I think this makes a little more sense.

Reply:
Great catch ! Interfaces tend to confuse me a bit so this was very helpful :)